src/services/billsService.js

```diff
@@ -2,5 +2,8 @@ import api from 'config/api';
 
 export default {
   getBills: () => api.get('listado_de_facturas'),
-  getLastBill: () => api.get('ultima_factura')
+  getLastBill: () => api.get('ultima_factura'),
+  postDigitalBill: email => api.post('factura_digital', email),
+  putDigitalBill: emailToUpdate => api.put('factura_digital', emailToUpdate),
```

Comment:
Generalmente, para aportarle semántica a nuestra funciones, no las solemos llamar (salvo en casos como el get o delete) por el verbo https. Te dejo un ejemplo de como llamaria yo a este servicio.
```suggestion
  updateDigitalBill: emailToUpdate => api.put('factura_digital', emailToUpdate),
```

Reply:
```javascript
  addDigitalBill: email => api.post('factura_digital', email),
  updateDigitalBill: emailToUpdate => api.put('factura_digital', emailToUpdate),
  removeDigitalBill: () => api.delete('factura_digital')
```